codepipeline-tf/modules/xc3_codepipeline/readme.md

```diff
@@ -0,0 +1 @@
+This pipeline has currently different stages to Source, Build, Deploy and destroy the code.
```

Comment:
please add all relevant detail in readme how you are deploying infra using pipeline what are the steps